evalml/tests/data_checks_tests/test_id_columns_data_check.py

```diff
@@ -186,85 +320,116 @@ def test_id_columns_strings():
         logical_types={
             "col_1_id": "categorical",
             "col_2": "categorical",
-            "Id": "categorical",
             "col_5": "categorical",
         },
     )
 
     id_cols_check = IDColumnsDataCheck(id_threshold=0.95)
     assert id_cols_check.validate(X) == [
         DataCheckWarning(
-            message="The first column 'col_1_id' is most likely to be the ID column. Columns 'Id', 'col_2', 'col_3_id' are also 95.0% or more likely to be an ID column",
+            message="The first column 'col_1_id' has a high likelihood of being the primary key. Columns 'col_2', 'col_3_id' are 95.0% or more likely to be an ID column",
             data_check_name=id_data_check_name,
             message_code=DataCheckMessageCode.HAS_ID_FIRST_COLUMN,
-            details={"columns": ["col_1_id", "Id", "col_2", "col_3_id"]},
+            details={"columns": ["col_1_id", "col_2", "col_3_id"]},
             action_options=[
                 DataCheckActionOption(
                     DataCheckActionCode.SET_FIRST_COL_ID,
                     data_check_name=id_data_check_name,
-                    metadata={"columns": ["col_1_id", "Id", "col_2", "col_3_id"]},
+                    metadata={"columns": ["col_1_id", "col_2", "col_3_id"]},
                 ),
             ],
         ).to_dict(),
     ]
 
-
-def test_id_cols_data_check_input_formats():
-    id_cols_check = IDColumnsDataCheck(id_threshold=0.8)
-
-    # test empty pd.DataFrame
-    assert id_cols_check.validate(pd.DataFrame()) == []
-
-    #  test Woodwork
-    ww_input = pd.DataFrame(np.array([[0, 1], [1, 2], [2, 3], [3, 4], [4, 5]]))
-    ww_input.ww.init()
-    assert id_cols_check.validate(ww_input) == [
+    X = X.rename(columns={"col_1_id": "ID"})
+    assert id_cols_check.validate(X) == [
         DataCheckWarning(
-            message="The first column '0' is most likely to be the ID column. Columns '1' are also 80.0% or more likely to be an ID column",
+            message="The first column 'ID' has a high likelihood of being the primary key. Columns 'col_2', 'col_3_id' are 95.0% or more likely to be an ID column",
             data_check_name=id_data_check_name,
             message_code=DataCheckMessageCode.HAS_ID_FIRST_COLUMN,
-            details={"columns": [0, 1]},
+            details={"columns": ["ID", "col_2", "col_3_id"]},
             action_options=[
                 DataCheckActionOption(
                     DataCheckActionCode.SET_FIRST_COL_ID,
                     data_check_name=id_data_check_name,
-                    metadata={"columns": [0, 1]},
+                    metadata={"columns": ["ID", "col_2", "col_3_id"]},
```

Comment:
How is this working exactly? How can we tell which columns the recommendation is to drop and which columns it recommends to mark primary?

Reply:
It is checking if the first column satisfies these two characteristics:

- It is either named 'ID' or its name ends with '_id'
- All of it's values are unique

If those characteristics are met, it suggests to mark as primary key. Otherwise, it suggests to drop columns

Reply:
In this case , the columns name was 'ID' and had the values ["a", "b", "c", "d"] which are all unique

Reply:
Right, I think the point of confusion comes in the metadata. When we look at the action code `SET_FIRST_COL_ID` and then look at the column metadata `['ID', 'col_2', ...]`, how do we know which of these is the col_id to set as the primary key? It would be clearer if the metadata had one value only when we need set the column so that we don't have to worry about confusion on later steps. The additional columns should be listed somewhere else.

Reply:
Oh okay in the case where the data check action code is `SET_FIRST_COL_ID` , the first column name in the metadata is always the column to set as the primary key and the other columns are the ones to drop. Would it be a good idea for me to change the metadata structure to something like `{ "primary_key": "ID", "drop": ['col_2', 'col_3_id']`

Reply:
Yea, this seems clearer to me! It makes more sense and will be much easier for developers down the line to understand

Reply:
Okay done!

Reply:
Ah, rather than using `drop` for the other columns, can you just continue to list them under `columns` value? I think that'd be fine rather than introducing a new value to check.